autoload/clap/provider/help_tags.vim

```diff
@@ -0,0 +1,65 @@
+" Author: Mark Wu <markplace@gmail.com>
+" Description: List the help tags.
+
+let s:save_cpo = &cpoptions
+set cpoptions&vim
+
+" Help tags support, ported from https://github.com/zeero/vim-ctrlp-help
+let s:help_tags_filecache = get(g:, 'clap_provider_help_tags_cache', fnamemodify($MYVIMRC, ':p:h') . '/clap_provider_help_tags_cache')
+let s:help_tags_memcache = []
+
+function! s:help_tags_source() abort
+    if(empty(s:help_tags_memcache))
```

Comment:
```suggestion
    if empty(s:help_tags_memcache)
```

Reply:
Ha, I forgot I am writing VIM not PHP. Done.